python/tank_vendor/shotgun_api3/shotgun.py

```diff
@@ -92,7 +92,7 @@
 
 # ----------------------------------------------------------------------------
 # Version
-__version__ = "3.0.33.dev"
+__version__ = "3.0.32.toolkit"
```

Comment:
As you can see, the release has been tagged as 3.0.32.toolkit, which is our own special build on top of 3.0.32.